src/blocks/common.blocks/search-form/_type/search-form_type_default.styl

```diff
@@ -0,0 +1,4 @@
+.search-form_type_default .search-form__input
```

Comment:
can be replaced with [input_width_available](https://ru.bem.info/libs/bem-components/v2.1.0/desktop/input/#width)